src/App.js

```diff
@@ -1,26 +1,120 @@
 import React from 'react';
-import logo from './logo.svg';
 import './App.css';
+import Header from './components/Header';
+import Footer from './components/Footer';
+import Main from './components/Main';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      value: '',
+      items: [],
+      selectedList: 0,
+      checked: false,
+    };
+
+    this.addToValue = (event) => {
+      this.setState({
+        value: event.target.value,
+      });
+    };
+
+    this.addToItems = () => {
+      if (this.state.value) {
+        this.setState(prevState => ({
+          items: [
+            ...prevState.items,
+            {
```

Comment:
Add `id` field. It is mandatory to have ID and it should be unique for each item. Imagine you have 2 or more items with the same value. This way you can't work with each of them separately (e.g. toggle status on click).
Suggestion: use `Date.now()` as id
Suggestion 2: use `random string` library: https://www.npmjs.com/package/randomstring

Reply:
![image](https://user-images.githubusercontent.com/29413892/59746485-b06dd280-927f-11e9-94b6-0f8e0968f0a9.png)